### PR TITLE
Sanitize css properties of style attributes

### PR DESCRIPTION
**Pull request by gerad**
https://github.com/microcosm-cc/bluemonday/issues/43

### CODE REVIEW

File: sanitize_test.go [hunk 1]
```diff
@@ -1514,3 +1514,69 @@ func TestTargetBlankNoOpener(t *testing.T) {
...
+			in:       `<div style="background: javascript:alert('xss');">no xss</div>`,
+			expected: `<div style="">no xss</div>`,
+		},
+		{
+			in:       `<div style="background: javascript:alert('xss')">no xss</div>`,
+			expected: `<div style="">no xss</div>`,
+		},
+		{
+			in:       `<div style="text-align;background: javascript:alert('xss')">no xss</div>`,
+			expected: `<div style="">no xss</div>`,
```
Comment (gdehmlow) [inline sanitize_test.go:1550]: Do we maybe just want to remove the style attribute if it's an empty string? i don't think it particularly matters but

**Comment by gdehmlow** [comment]
Looks good to me

File: sanitize_test.go (hunk 1 above)
Comment (gerad) [reply sanitize_test.go:1550]: Yeah, there's a different option that should remove it… but it doesn't seem to work: https://github.com/sagansystems/bluemonday/blob/master/policy.go#L154

Comment (gerad) [reply sanitize_test.go:1550]: I didn't want to try and fix the bug in this PR. When I was searching through the code, it looked like it was just globally broken, not broken by this PR.

**Pull request merged**